packages/clay-navigation/src/Breadcrumb.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+import warning from 'warning';
+import {BreadcrumbEllipsis} from './BreadcrumbEllipsis';
+import {BreadcrumbItem, IBreadcrumbItem} from './BreadcrumbItem';
+
+interface IProps extends React.HTMLAttributes<HTMLOListElement> {
+	ellipsisBuffer?: number;
+
+	ellipsisProps?: Object;
+
+	items: Array<IBreadcrumbItem>;
+
+	spritemap: string;
+}
+
+const getBufferList = (
+	items: Array<IBreadcrumbItem>,
```

Comment:
Generally I think we tend to opt for `IBreadcrumbItem[]` for array types. I may be wrong, or we might need to add an eslint rule for this.

Reply:
I think they are just different ways of doing this, I like `Array<...>` because it is more explicit to me and goes along with the other syntaxes like `React.FunctionComponent<...>`.

Reply:
@matuzalemsteles yeah thats fine, we likely should just pick one or the other and stick with it for consistency

Reply:
Sure, I think we can only use `Array<...>` for consistency, is there any way to add that in lint?

Reply:
Yeah I think we can add, https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/array-type.md#generic

Reply:
I vote for Array<anything> too :)

Reply:
https://github.com/liferay/clay/pull/2169